browser/src/map/Clipboard.js

```diff
@@ -862,20 +859,16 @@ L.Clipboard = L.Class.extend({
 	},
 
 	_warnCopyPaste: function() {
-		var self = this;
 		var msg;
 		if (window.mode.isMobile() || window.mode.isTablet()) {
 			msg = _('<p>Please use the copy/paste buttons on your on-screen keyboard.</p>');
 		} else {
 			msg = _('<p>Your browser has very limited access to the clipboard, so use these keyboard shortcuts:</p><table class="warn-copy-paste"><tr><td><kbd>Ctrl</kbd><span class="kbd--plus">+</span><kbd>C</kbd></td><td><kbd>Ctrl</kbd><span class="kbd--plus">+</span><kbd>X</kbd></td><td><kbd>Ctrl</kbd><span class="kbd--plus">+</span><kbd>V</kbd></td></tr><tr><td>Copy</td><td>Cut</td><td>Paste</td></tr></table>');
 			msg = L.Util.replaceCtrlAltInMac(msg);
 		}
-		vex.dialog.alert({
-			unsafeMessage: msg,
-			callback: function () {
-				self._map.focus();
-			}
-		});
+
+		this._map.uiManager.showInfoModal('copy paste warning');
+		document.getElementById('copy paste warning').innerHTML = msg;
```

Comment:
element id with spaces? is that allowed?

Reply:
yep, that's allowed.

Reply:
how to write CSS rule for that for example? :)

Reply:
Let me change that in case we need special cases, thanks :)